_sass/components/partners/_partners.scss

```diff
@@ -76,4 +97,22 @@
 .usa-process-list__item::before {
     background-color: #0071BB !important;
     border-color: #0071BB !important;
+}
+
+.usa-accordion__heading h4 {
+    font-family: $monospace-font-family;
```

Comment:
For some reason the very first h4 is still not the right font??
<img width="586" alt="Screen Shot 2021-12-29 at 8 56 06 AM" src="https://user-images.githubusercontent.com/25647797/147669537-1c485fb9-1978-4caa-b688-7410ebf8c555.png">